extensions/binding/pom.xml

```diff
@@ -34,6 +34,8 @@
     <module>org.eclipse.smarthome.binding.wemo</module>
     <module>org.eclipse.smarthome.binding.wemo.test</module>
     <module>org.eclipse.smarthome.binding.yahooweather</module>
+    <module>org.eclipse.smarthome.binding.dmx</module>
+    <module>org.eclipse.smarthome.binding.dmx.test</module>
   </modules>
 
-</project>
+</project>
```

Comment:
And also to https://github.com/eclipse/smarthome/blob/master/docs/_includes/documentation-menu.html#L23-L33?